torchx/tracker/api.py

```diff
@@ -281,7 +281,7 @@ def run_from_env() -> AppRun:
             for tracker in trackers:
                 tracker.add_source(torchx_job_id, parent_run_id, artifact_name=None)
 
-        return AppRun(id=torchx_job_id, backends=trackers_from_environ())
+        return AppRun(id=torchx_job_id, backends=trackers)
```

Comment:
This should really be in a different PR -- though it's small enough

Reply:
yea... you are right, I should've done this in a different PR... just got a bit lazy at 3am... will make sure I split the PRs going forward.